src/chains/tezos/options/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../../../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "lib"
+    "outDir": "lib",
+    "composite": true
```

Comment:
eventually this will be used in other packages... so let's include `composite: true` now, since it will be required later.